cinema/serializers.py

```diff
@@ -33,7 +33,17 @@ class Meta:
 class MovieSerializer(serializers.ModelSerializer):
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = ("id",
+                  "title",
+                  "description",
+                  "duration",
+                  "genres",
+                  "actors",)
+
+
+class MovieImageSerializer(MovieSerializer):
+    model = Movie
+    fields = ("id", "image")
```

Comment:
The `MovieImageSerializer` should inherit from `serializers.ModelSerializer` instead of `MovieSerializer`. This is necessary to properly define the fields and model for this serializer.